### ISSUE TITLE
Mozilla FF & Chrome & Me Issues

### ISSUE THREAD

**Issue by angeldjac** [darktrojan/newtabtools#376]
I am NOT a techie -- I just miss my tabs that I created before I made the mistake of updating FF to v57!
Now, all I get is a blank screen (lost all of my pretty tabs and only have a blank screen)... 

SO ... I figured I'd ditch FF and switch to Chrome ...
The tabs show up as I add them but I can't get ANY thumbnails/images in the tab options 
AND
when I "right-click" my mouse to edit/add a tab nothing happens ... I have to use the "gear" icon to add or change anything ...

someone PLEASE help me and tell me what I am doing wrong!!

thanks!

**Comment by artworksmetal** [comment]
Yeah, I feel your pain. Whether Firefox or Chrome, it's kind of painful now. See my thread [https://github.com/darktrojan/newtabtools/issues/374](url) .
In Chrome, the image files are stored in the following folder:

"C:\Users\[Your User Name]\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_gfonmeedigdbjpgpeojioadpimflbogh_0.indexeddb.blob\1\00\

The long gobbledygook after extension will be different for you. 
Then, to make matters worse, the images are PNG files without the extension. The Open Dialogue defaults to looking for image files, so you have to change that to All Files to see the actual image files.
 A good way to deal with that is to have Irfanview. It will let you open the extensionless file. It will add the file extension for you and you can rename it easily within the program (hit F2).
As I suggested in my thread, backup and restore often if you are making a complicated grid and renaming files.
I almost exclusively use Firefox, so I'm not sure how well Chrome will handle this. But I hope this helps get you on the right track.

**Comment by angeldjac** [comment]
thanx for the quick response ... 
IF i want to continue to use FF? (_old habits with old people - blah blah blah_) what do I need to do to get NewTabs to even show up?
(ps - i LOVE how we can backup/restore now!!)

**Comment by artworksmetal** [comment]
yeah, I still prefer FF significantly over Chrome. I'm not sure what you mean about NTT not showing up. Are you talking about the tiles? Please explain more and post a screen print if that will help.
Here's what mine is looking like now. But trust me, it's a good thing I am retired and have time to screw with this.
![ntt_screen](https://user-images.githubusercontent.com/31712900/34954109-374cc824-f9ed-11e7-9350-b98349e44a2e.png)

**Comment by angeldjac** [comment]
i can't do a screen shot because it's on my laptop at home  ... i am guessing this is a ME issue because i have the same FF version on my work computer and it's fine . . .
in a nutshell, the screen is all white and i have no options to add tabs or even right-click and use the tool.
my plan is to take the "zip file" from my work computer and use it on my home computer and see if i can get the tabs to display ... 

i'll let you know if that solves the problem in about 6 hrs from now ... 

_**THANX again!!!**_   awesome tool, awesome support !!

**Comment by darktrojan** [comment]
> I am NOT a techie -- I just miss my tabs that I created before I made the mistake of updating FF to v57!
> Now, all I get is a blank screen (lost all of my pretty tabs and only have a blank screen)...

If New Tab Tools is installed and working properly you should see it in new tabs. If not you might find some other extension is trying to control the new tab page. If your *home page* was New Tab Tools before updating to 57, you'll have to change it (first get NTT working, then drag the icon from the address bar onto the home button).

> The tabs show up as I add them but I can't get ANY thumbnails/images in the tab options

Should just work, but the Chrome version is pretty new and there could be bugs I haven't come across yet. I'm not sure what you mean by this so more info about what you're trying to do would be helpful.

> when I "right-click" my mouse to edit/add a tab nothing happens ... I have to use the "gear" icon to add or change anything ...

Chrome doesn't have the right-click menu, I can't fix that.

**Comment by angeldjac** [comment]
using FF:

here are my add-ons (same as before the FF update):
![image](https://user-images.githubusercontent.com/35415122/35001820-b74ecd7e-fab5-11e7-9ae9-e7df69f62353.png)

here is what I see when I open a New Tab:
![image](https://user-images.githubusercontent.com/35415122/35001898-f2aa186a-fab5-11e7-95d9-e840957dea5e.png)

I have used the "Pin to the New Tab page" and "Capture Thumbnail" and nothing happens . . .

What do I try next?  I can't even import my backup NTT file from my work computer . . .

**Comment by angeldjac** [comment]
ah geez . . .NOW it's working ... wtheck!

closing this thread -- thanks for your patience with ME !!!

**Comment by darktrojan** [comment]
That's the Firefox default new tab… why it's not registered the extension's new tab, who knows? :-)

**Comment by artworksmetal** [comment]
My guess is under the options @angeldjac has "Blank Page" selected.
![nthome](https://user-images.githubusercontent.com/31712900/35014060-6803fdf2-fadd-11e7-9b17-b37d2588b07c.jpg)
if you want NTT to come up when you open Firefox, then do the following:
Open Firefox
Click on a new tab (+ sign) twice to open NTT twice
Close the blank new tab page.
Go to the right hand NTT page and select Tools/Options from the menu.
Select "Show your home pages" and click on the "Use Current Pages" button.
